### ISSUE TITLE
Cannot read property 'displaySets' of undefined

### ISSUE THREAD

**Issue by jwitos** [OHIF/Viewers#2569]
Using latest OHIF docker image.

In some cases, certain sequences, specifically sagittal / coronal MPR reconstructions, do not open in the viewer. They are listed in the series list pane with correct series number and no thumbnail. When clicking on one of those sequences, the following errors show up:

```
TypeError: Cannot read property 'displaySets' of undefined
    at Object.isDisabled (app.bundle.27574fe61efb99f6fb02.js:241234)
    at ConnectedStandaloneRouting~IHEInvokeImageDisplay~ViewerRouting.bundle.51152c80f2a231a39ce2.js:372
    at Array.forEach (<anonymous>)
    at ConnectedStandaloneRouting~IHEInvokeImageDisplay~ViewerRouting.bundle.51152c80f2a231a39ce2.js:363
    at Array.forEach (<anonymous>)
    at ToolbarRow.updateButtonGroups (ConnectedStandaloneRouting~IHEInvokeImageDisplay~ViewerRouting.bundle.51152c80f2a231a39ce2.js:360)
    at ToolbarRow.componentDidUpdate (ConnectedStandaloneRouting~IHEInvokeImageDisplay~ViewerRouting.bundle.51152c80f2a231a39ce2.js:440)
    at ik (app.bundle.27574fe61efb99f6fb02.js:203048)
    at exports.unstable_runWithPriority (app.bundle.27574fe61efb99f6fb02.js:203124)
    at fg (app.bundle.27574fe61efb99f6fb02.js:202917)
```
```
ToolbarRow Error Boundary TypeError: Cannot read property 'displaySets' of undefined
    at Object.isDisabled (app.bundle.27574fe61efb99f6fb02.js:241234)
    at ConnectedStandaloneRouting~IHEInvokeImageDisplay~ViewerRouting.bundle.51152c80f2a231a39ce2.js:372
    at Array.forEach (<anonymous>)
    at ConnectedStandaloneRouting~IHEInvokeImageDisplay~ViewerRouting.bundle.51152c80f2a231a39ce2.js:363
    at Array.forEach (<anonymous>)
    at ToolbarRow.updateButtonGroups (ConnectedStandaloneRouting~IHEInvokeImageDisplay~ViewerRouting.bundle.51152c80f2a231a39ce2.js:360)
    at ToolbarRow.componentDidUpdate (ConnectedStandaloneRouting~IHEInvokeImageDisplay~ViewerRouting.bundle.51152c80f2a231a39ce2.js:440)
    at ik (app.bundle.27574fe61efb99f6fb02.js:203048)
    at exports.unstable_runWithPriority (app.bundle.27574fe61efb99f6fb02.js:203124)
    at fg (app.bundle.27574fe61efb99f6fb02.js:202917) 
    in ToolbarRow
    in WrappedComponent
    in WrappedComponent
    in WrappedComponent
    in withI18nextTranslation(WrappedComponent)
    in ErrorBoundary
    in OHIFErrorBoundary
    in ErrorBoundaryDialog
    in Viewer
    in WrappedComponent
    in ConnectFunction
    in ViewerRetrieveStudyData
    in ConnectFunction
    in ViewerRouting
    in Unknown
    in ErrorBoundary
    in OHIFErrorBoundary
    in Transition
    in CSSTransition
    in Route
    in OHIFStandaloneViewer
    in ConnectFunction
    in withRouter(Connect(OHIFStandaloneViewer))
    in Unknown
    in Unknown
    in ModalProvider
    in DialogProvider
    in SnackbarProvider
    in LoggerProvider
    in Router
    in BrowserRouter
    in I18nextProvider
    in AppProvider
    in Provider
    in ErrorBoundary
    in OHIFErrorBoundary
    in App
```

Would appreciate any tips on how to debug this further.

**Comment by sedghi** [comment]
Can you clarify what you mean by MPR being in the series list and not having thumbnail? MPR is a visualization technique and series can be reconstructed in other views, but that doesn't mean they will appear in the series list on the left

**Comment by jwitos** [comment]
@sedghi it's been a couple of months so I can't really offer any more debugging at this point. You're right that it was not MPR, what I meant is that it was a reconstruction in sagittal / coronal axis (I don't know if it was acquired or reconstructed in these axes), but they were separate series. I was able to open and see them in other DICOM viewers. In OHIF, they showed up in the series list but with no thumbnail, and after I tried to open them the error above showed up.
